static/beta/stage/navigation/settings-navigation.json

```diff
@@ -36,7 +36,7 @@
             "permissions": [
                 {
                     "method": "featureFlag",
-                    "args": ["platform.sources.integrations", false]
+                    "args": ["platform.sources.integrations", true]
```

Comment:
This will surface both `sources` and `sources-integrations` to navigation. It should be false.